soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -28,6 +28,10 @@ config DISPLAY_MCUX_ELCDIF
 	default y if HAS_MCUX_ELCDIF
 	depends on DISPLAY
 
+config MIPI_DSI_MCUX
+	default y if HAS_MCUX_MIPI_DSI
+	depends on MIPI_DSI
```

Comment:
note: this should not be needed if the driver Kconfig defaults based on DT compatible status. You can find examples in a few areas, e.g. pinctrl drivers.

Reply:
Thanks for pointing this, I have made the change